create.cjs

```diff
@@ -701,3 +676,84 @@ function generateDonwloadStats() {
     ).join('\n')
   );
 }
+
+/**
+ * @param {VirtualPackage} pkg
+ */
+async function writePackage(pkg) {
+  await fsPromises.mkdir(pkg.path, { recursive: true });
+  let hasChanged = false;
+
+  const promises = [];
+
+  const existingFiles = await fg(['**/*'], {
```

Comment:
Using `fast-glob` just for listing files could be an overkill. I prefer using Isaacs's `path-scurry` package: https://github.com/isaacs/path-scurry. It is lightweight and performant.

Reply:
Interesting lib, thanks for sharing :)